src/fides/core/config/utils.py

```diff
@@ -4,6 +4,29 @@
 DEFAULT_CONFIG_PATH_ENV_VAR = "FIDES__CONFIG_PATH"
 
 
+def replace_config_value(
+    fides_directory_location: str, key: str, old_value: str, new_value: str
+) -> None:
+    """Use string replacment to update a value in the fides.toml"""
+
+    # This matches the logic used in `docs.py`
+    fides_dir_name = ".fides"
+    fides_dir_path = f"{fides_directory_location}/{fides_dir_name}"
+    config_file_name = "fides.toml"
+    config_path = f"{fides_dir_path}/{config_file_name}"
+
+    with open(config_path, "r", encoding="utf8") as config_file:
+        previous_config = config_file.read()
+        new_config = previous_config.replace(
+            f"{key} = {old_value}", f"{key} = {new_value}"
+        )
+
+    with open(config_path, "w", encoding="utf8") as config_file:
+        config_file.write(new_config)
+
+    print(f"Config key: {key} value changed: {old_value} -> {new_value}")
```

Comment:
Would this print sensitive values to stdout?

Reply:
yes, but currently this is specifically only designed to be used in this one specific situation. Currently there is no code that calls this that would have sensitive info in it